### PR TITLE
Fixing the link to kubernetes registry

### PR DESCRIPTION
**Pull request by Pensu**
The old link doesn't work.

[skip ci]

### CODE REVIEW

File: Documentation/filesystem.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ If the Rook cluster has more than one filesystem and the application pod is sche
 
 ## Consume the Shared File System: Toolbox
 
-Once you have pushed an image to the registry (see the [instructions](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/registry) to expose and use the kube-registry), verify that kube-registry is using the filesystem that was configured above by mounting the shared file system in the toolbox pod. See the [Direct Filesystem](direct-tools.md#shared-filesystem-tools) topic for more details.
+Once you have pushed an image to the registry (see the [instructions](https://github.com/kubernetes/kubernetes/tree/7ef585be224dae4ec5deae0f135653116e21a6e0/cluster/addons/registry) to expose and use the kube-registry), verify that kube-registry is using the filesystem that was configured above by mounting the shared file system in the toolbox pod. See the [Direct Filesystem](direct-tools.md#shared-filesystem-tools) topic for more details.
```
Comment (travisn) [inline Documentation/filesystem.md]: thanks for catching this. How recent is that hash? What if we use the `release-1.9` tag as the last tag where it was available?

**Comment by Pensu** [comment]
@travisn Yeah, makes sense, updated. PTAL.

**Comment by jbw976** [comment]
Do we know why the documentation in the kubernetes repo was deleted?  is this example not applicable anymore starting with 1.10?

**Comment by jbw976** [comment]
it was removed this commit, but it doesn't say why: https://github.com/kubernetes/kubernetes/commit/d6918bbbc0402fc81a53479f4b61b836d7c33a29

**Pull request merged**